### ISSUE TITLE
Support for other workflow actions

### ISSUE THREAD

**Issue by molivasdat** [nektos/act#400]
act -l does not show the job if the event is not push or pull (the **on:** stanza)
If I set this to [workflow_dispatch](https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#manual-events), act will not see the job.
If I uncomment push,
 act will see the job and be able to run it locally.
```# This is a basic workflow to help you get started with Actions
name: Liquibase_Simple

# Controls when the action will run. Triggers the workflow on push or pull request
# events but only for the master branch
on:
  workflow_dispatch:
  #push:
# A workflow run is made up of one or more jobs that can run sequentially or in parallel
jobs:
  SimpleCLI:
    runs-on: ubuntu-latest
    steps: 
    # Runs a set of commands using the runners shell 
    - uses: actions/checkout@v2
    - name: What OS is running
      run: uname -a
```

**Comment by revmischa** [comment]
I would love it if this worked as well!

**Comment by edtan** [comment]
The `act --list` issue is a duplicate of #317.  You will need to specify the event type: `act workflow_dispatch --list`.

If you want to run the `workflow_dispatch` event, you need to explicitly specify the event type:
```
act workflow_dispatch
```

**Comment by molivasdat** [comment]
Thanks so much so updating the readme and the update to examples to show other event types!  Great project!
